modules/candidate_parameters/jsx/FamilyInfo.js

```diff
@@ -9,22 +13,28 @@ let FamilyInfo = React.createClass({
       isLoaded: false,
       loadedData: 0,
     };
-  },
-  componentDidMount: function() {
+    this.setFormData = this.setFormData.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.showAlertMessage = this.showAlertMessage.bind(this);
+    this.deleteFamilyMember = this.deleteFamilyMember.bind(this);
+  }
+
+  componentDidMount() {
     let that = this;
     $.ajax(
-        this.props.dataURL,
+      this.props.dataURL,
```

Comment:
could move ajax call out into a fetchData() function and then remove `let that = this;`